Content.Shared/GameObjects/Components/Chemistry/ISolutionInteractionsComponent.cs

```diff
@@ -64,6 +64,11 @@ Solution Draw(ReagentUnit amount)
         bool CanRefill => false;
         ReagentUnit RefillSpaceAvailable => ReagentUnit.Zero;
 
+        /// <summary>
+        ///     The amount that will transfer if something is spilled on the container.
+        /// </summary>
+        ReagentUnit Spillability => ReagentUnit.Zero;
```

Comment:
"Spillability" sounds more like it is the thing being *spilled*, not being spilled _into_.

A better name might be like `SpillRefillMax`.